src/Iot.Device.Bindings/Iot.Device.Bindings/Iot.Device.Bindings.sln

```diff
@@ -0,0 +1,101 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
We tipically don't checkin solutions into the repo with the only exception of the main library.

Reply:
It was a mistake, I removed it again